txt2tags.py

```diff
@@ -1189,6 +1195,60 @@ def getTags(config):
             # if possible: http://www.wikicreole.org/wiki/Placeholder
         },
+            # regular markdown: http://daringfireball.net/projects/markdown/syntax
+            # markdown extra:   http://michelf.com/projects/php-markdown/extra/
+        "md": {
+            "title1"               : "# \a "         ,
+            "title2"               : "## \a "        ,
+            "title3"               : "### \a "       ,
+            "title4"               : "#### \a "      ,
+            "title5"               : "##### \a "     ,
+            "blockVerbLine"        : "    "          ,
+            "blockQuoteLine"       : "> "            ,
+            "fontMonoOpen"         : "`"             ,
+            "fontMonoClose"        : "`"             ,
+            "fontBoldOpen"         : "**"            ,
+            "fontBoldClose"        : "**"            ,
+            "fontItalicOpen"       : "*"             ,
+            "fontItalicClose"      : "*"             ,
+            "fontUnderlineOpen"    : ""              ,
+            "fontUnderlineClose"   : ""              ,
+            "fontStrikeOpen"       : "~~"            ,
+            "fontStrikeClose"      : "~~"            ,
+            # Lists
+            #"listOpenCompact"     : "*"             ,
+            "listItemLine"         : " "             ,
+            "listItemOpen"         : "*"             ,
+            #"numlistItemLine"     : "1."            ,
+            "numlistItemOpen"      : "1."            ,
+            "deflistItem1Open"     : ": "            ,
+            #"deflistItem1Close"     : ":"           ,
+            #"deflistItem2LineOpen"  : "::"          ,
+            #"deflistItem2LineClose" : ":"           ,
+            # Verbatim block
+            #"blockVerbOpen"        : ""             ,
+            #"blockVerbClose"       : ""             ,
+            "bar1"                 : "---"           ,
+            "bar2"                 : "---"           ,
+            # URL, email and anchor
+            "url"                   : "\a"           ,
+            "urlMark"               : "[\a](\a)"     ,
+            "email"                 : "\a"           ,
+            #"emailMark"             : "[[\a -> mailto:\a]]",
+            #"anchor"                : "[[#\a]]\n"   ,
+            # Image markup
+            "img"                   : "![](\a)"      ,
+            #"imgAlignLeft"         : "{{\a }}"      ,
+            #"imgAlignRight"        : "{{ \a}}"      ,
+            #"imgAlignCenter"       : "{{ \a }}"     ,
+            # Table attributes
+            "tableTitleRowOpen"    : "| "            ,
+            "tableTitleRowClose"   : "|\n|---------------|"            ,
+            "tableTitleCellSep"    : " |"            ,
+            "tableRowOpen"         : "|"             ,
+            "tableRowClose"        : "|"             ,
+            "tableCellSep"         : " |"            ,
+        },
```

Comment:
You'll have to run black (https://github.com/psf/black) on txt2tags.py for the syntax checks to pass.

Reply:
done